packages/components/src/sandbox/index.native.js

```diff
@@ -215,7 +215,7 @@ function Sandbox( {
 					<title>{ title }</title>
 					<meta
 						name="viewport"
-						content="width=device-width, initial-scale=1"
+						content={ `width=device-width, initial-scale=1, ${ viewportProps }` }
```

Comment:
I verified that an  extra comma is fine in the case where someone prefixes `viewportProps` with a comma like  ",  user-scalable=0"